tests/src/test/fdc3.broadcast.ts

```diff
@@ -32,135 +30,155 @@ export default () =>
     });
 
     describe("User channels", () => {
+      beforeEach(async () => {
+        clearTimeout(timeout);
+        await unsubscribeListeners();
+        await window.fdc3.leaveCurrentChannel();
+        resetChannelsAppContext();
+      });
+
       afterEach(async () => {
+        clearTimeout(timeout);
         await broadcastSystemChannelCloseWindow();
       });
 
       it("Should receive context when adding a listener then joining a user channel before app B broadcasts context to the same channel", async () => {
-        return new Promise(async (resolve) => {
+        const errorMessage = `\r\nSteps to reproduce:\r\n- Add fdc3.instrument context listener to app A\r\n- App A joins channel 1\r\n- App B joins channel 1\r\n- App B broadcasts fdc3.instrument context${documentation}`;
```

Comment:
minor point, as you're creating a string via [template literal syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals), you don't need to add escaped newlines, you can make this a multi-line string directly.